twisted/twisted/plugins/tunnel_helper_plugin.py

```diff
@@ -130,9 +162,7 @@ def start_tribler(self):
         config.set_enable_torrent_search(False)
         config.set_enable_channel_search(False)
         self.session = Session(config)
-        upgrader = self.session.prestart()
-        while not upgrader.is_done:
-            time.sleep(0.1)
+        self.session.prestart()
```

Comment:
This would not be enough, the upgrader opens the database which needs to be closed I think.